docs/source/en/generation_strategies.md

```diff
@@ -416,16 +416,6 @@ Assisted decoding assumes the main and assistant models have the same tokenizer,
 Currently, only greedy search and sampling are supported with assisted decoding, and assisted decoding doesn't support batched inputs.
 To learn more about assisted decoding, check [this blog post](https://huggingface.co/blog/assisted-generation).
 
-#### Universal Assisted Decoding
```

Comment:
The current version of the docs had the normal assisted generation docs under `Universal Assisted Decoding` [modification to the original technique to support different tokenizers]

Most of the diff is to isolate `Universal Assisted Decoding`